packages/plugin-node/src/services/image.ts

```diff
@@ -97,11 +98,13 @@ export class ImageDescriptionService
 
             if (model === models[ModelProviderName.LLAMALOCAL]) {
                 await this.initializeLocalModel();
+            } else if (model === models[ModelProviderName.ANTHROPIC]) {
+                this.modelId = "claude-3-haiku-20240307";
+                this.device = "cloud";
             } else {
                 this.modelId = "gpt-4o-mini";
                 this.device = "cloud";
```

Comment:
The model selection logic lacks a default case for unrecognized models. This could lead to unexpected behavior if a new model is added but not handled. Consider adding a default case that logs an error or throws an exception to ensure that all models are accounted for.